_patternlab-files/partials/patternNav.twig

```diff
@@ -0,0 +1,60 @@
+<ul class="patterns-menu sg-nav">
+    <li class="sg-nav-"><a class="sg-acc-handle">base</a>
+        <ul class="sg-acc-panel">
+
+            <li class="sg-nav-"><a class="sg-pop" href="app/bower_components/pattern-library/patterns/base/blockquote">Blockquote</a></li>
```

Comment:
@scottnath , actually this template should contain @rebmullin 's work for patterns. Can you link me to that file? So, I can put it here and apply styling.